### PR TITLE
Improve memory store best practices page

### PR DESCRIPTION
**Pull request by Someon1e**
## Changes

- Renamed memoryStore variable to MemoryStoreService
- Used `game:GetService("Players"):GetPlayers()` instead of `game.Players:GetChildren()`
- Remove use of `pairs`
- Used `count // 4` instead of `math.floor(count / 4)`
- Use tabs instead of spaces and correct the indentation

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/cloud-services/memory-stores/best-practices.md [hunk 1]
```diff
@@ -34,13 +34,13 @@ Use a helper function to get the correct sorted map from an item key. This way,
 
 ```lua title='Sharding a Sorted Map'
 -- Initialize the MemoryStore Service
-local memoryStore = game:GetService("MemoryStoreService")
+local MemoryStore = game:GetService("MemoryStoreService")
```
Comment (IgnisRBX) [inline content/en-us/cloud-services/memory-stores/best-practices.md:37]:
```suggestion
local MemoryStoreService = game:GetService("MemoryStoreService")
```

File: content/en-us/cloud-services/memory-stores/best-practices.md [hunk 2]
```diff
@@ -34,13 +34,13 @@ Use a helper function to get the correct sorted map from an item key. This way,
...
 
 -- Create your Sorted Map buckets
-local sm_AtoG = memoryStore:GetSortedMap("AtoG")
-local sm_HtoM = memoryStore:GetSortedMap("HtoM")
-local sm_NtoT = memoryStore:GetSortedMap("NtoT")
-local sm_UtoZ = memoryStore:GetSortedMap("UtoZ")
+local sm_AtoG = MemoryStore:GetSortedMap("AtoG")
+local sm_HtoM = MemoryStore:GetSortedMap("HtoM")
+local sm_NtoT = MemoryStore:GetSortedMap("NtoT")
+local sm_UtoZ = MemoryStore:GetSortedMap("UtoZ")
```
Comment (IgnisRBX) [inline content/en-us/cloud-services/memory-stores/best-practices.md:43]:
```suggestion
local sm_AtoG = MemoryStoreService:GetSortedMap("AtoG")
local sm_HtoM = MemoryStoreService:GetSortedMap("HtoM")
local sm_NtoT = MemoryStoreService:GetSortedMap("NtoT")
local sm_UtoZ = MemoryStoreService:GetSortedMap("UtoZ")
```

File: content/en-us/cloud-services/memory-stores/best-practices.md [hunk 3]
```diff
@@ -83,13 +83,13 @@ One solution is to use a revolving queue, which means creating multiple queues a
 
 ```lua title='Sharding a Queue'
 -- Initialize the MemoryStore Service
-local memoryStore = game:GetService("MemoryStoreService")
+local MemoryStore = game:GetService("MemoryStoreService")
```
Comment (IgnisRBX) [inline content/en-us/cloud-services/memory-stores/best-practices.md:86]:
```suggestion
local MemoryStoreService = game:GetService("MemoryStoreService")
```

File: content/en-us/cloud-services/memory-stores/best-practices.md [hunk 4]
```diff
@@ -83,13 +83,13 @@ One solution is to use a revolving queue, which means creating multiple queues a
...
 
 -- Create your Queues
-local q1 = memoryStore:GetQueue("q1")
-local q2 = memoryStore:GetQueue("q2")
-local q3 = memoryStore:GetQueue("q3")
-local q4 = memoryStore:GetQueue("q4")
+local q1 = MemoryStore:GetQueue("q1")
+local q2 = MemoryStore:GetQueue("q2")
+local q3 = MemoryStore:GetQueue("q3")
+local q4 = MemoryStore:GetQueue("q4")
```
Comment (IgnisRBX) [inline content/en-us/cloud-services/memory-stores/best-practices.md:92]:
```suggestion
local q1 = MemoryStoreService:GetQueue("q1")
local q2 = MemoryStoreService:GetQueue("q2")
local q3 = MemoryStoreService:GetQueue("q3")
local q4 = MemoryStoreService:GetQueue("q4")
```

File: content/en-us/cloud-services/memory-stores/best-practices.md [hunk 5]
```diff
@@ -151,7 +151,7 @@ end
 
 -- Write some code!
 
-for i,player in pairs(game.Players:GetChildren()) do
+for i,player in game:GetService("Players"):GetPlayers() do
```
Comment (IgnisRBX) [inline content/en-us/cloud-services/memory-stores/best-practices.md:154]: Nice catch! Any other place you see `pairs()` being used when it's no longer needed, please feel free to submit a PR for it.

File: content/en-us/cloud-services/memory-stores/best-practices.md (hunk 1 above)
Comment (IgnisRBX) [reply content/en-us/cloud-services/memory-stores/best-practices.md:37]: Just matching the variable name to the service name, which we commonly like to do in docs.

**Review by IgnisRBX — APPROVED** [review]

**Comment by bUnsyv-5mamsy-cisfed** [comment]
Donate me 1000 robux you will earn 10000000 robux right now

**Pull request merged**